### PR TITLE
CR-1051866 : add base address to xbutil query

### PR DESCRIPTION
**Pull request by saifuddin-xilinx**
1. Added base address into xbutil query 
2. Added base address in xbutil2 report memory
Final Output :
**xbutil query:**
Memory Status
image

**xbutil2 memory :**
image

JSON output:
image

### CODE REVIEW

File: src/runtime_src/core/pcie/tools/xbutil/xbutil.h [hunk 1]
```diff
@@ -650,10 +650,14 @@ class device {
             std::stringstream ss(mm_buf[i]);
             ss >> memoryUsage >> boCount;
 
+	    std::stringstream ss_base_addr;
+	    ss_base_addr << "0x" << std::hex << map->m_mem_data[i].m_base_address;
```
Comment (chvamshi-xilinx) [inline src/runtime_src/core/pcie/tools/xbutil/xbutil.h:654]: fix indentation

File: src/runtime_src/core/pcie/tools/xbutil/xbutil.h [hunk 2]
```diff
@@ -650,10 +650,14 @@ class device {
...
+
             ptMem.put( "type",      str );
             ptMem.put( "temp",      (i >= temp_size) ? XCL_NO_SENSOR_DEV : temp[i]);
             ptMem.put( "tag",       map->m_mem_data[i].m_tag );
             ptMem.put( "enabled",   map->m_mem_data[i].m_used ? true : false );
+	    ptMem.put( "base_addr", ss_base_addr.str());
```
Comment (chvamshi-xilinx) [inline src/runtime_src/core/pcie/tools/xbutil/xbutil.h:660]: fix indentation

**Review by chvamshi-xilinx — APPROVED** [review]
Looks good except indentation

**Review by rozumx — APPROVED** [review]
Thank you for making these changes.  

Minor Request
Would you be so kind as to update the Pull Request comment indicating the new:
1) JSON output
2) Text output

This is for future developers.

**Pull request merged**